web/api/api.py

```diff
@@ -200,7 +200,7 @@ def update_plugin_category(slug, category):
     if not plugin:
         return api_util.api_not_found('No plugin with slug %s' % slug)
 
-    if not category in [c['id'] for c in get_all_categories_cached()]:
+    if category not in {c['id'] for c in get_all_categories_cached()}:
```

Comment:
Got it! I think I must have been confused about those two for some time now! Thanks pointing it out and thanks again for the correction.

Updated.